storage/innobase/xtrabackup/src/backup_copy.cc

```diff
@@ -1449,7 +1449,34 @@ bool backup_start(Backup_context &context) {
   xb_mysql_query(mysql_connection, "FLUSH NO_WRITE_TO_BINLOG BINARY LOGS",
                  false);
 
-  context.log_status = log_status_get(mysql_connection);
+  lsn_t page_tracking_start_lsn = 0;
+
+  if (opt_page_tracking &&
+      xb_start_page_tracking(mysql_connection, &page_tracking_start_lsn)) {
+    msg("xtrabackup: pagetracking is started on the server with LSN" LSN_PF
+        "\n",
+        page_tracking_start_lsn);
+  };
+
+  auto log_status = log_status_get(mysql_connection);
+
+  if (opt_page_tracking && page_tracking_start_lsn != 0) {
+    while (true) {
+      if (log_status.lsn_checkpoint >= page_tracking_start_lsn) {
+        msg("pagetracking: Checkpoint lsn is " LSN_PF
+            " and page_tracking lsn is " LSN_PF " lsn \n  ",
+            log_status.lsn_checkpoint, page_tracking_start_lsn);
+        break;
+      } else {
+        std::this_thread::sleep_for(std::chrono::seconds(1));
+        msg("pagetracking: waiting till checkpoint lsn " LSN_PF
+            " increased to page_tracking " LSN_PF " lsn \n  ",
+            log_status.lsn_checkpoint, page_tracking_start_lsn);
```

Comment:
```suggestion
        msg_ts("pagetracking: waiting for 1 second till checkpoint lsn " LSN_PF
            " increased to page_tracking " LSN_PF " lsn \n  ",
            log_status.lsn_checkpoint, page_tracking_start_lsn);
            std::this_thread::sleep_for(std::chrono::seconds(1));
```
First add the log message, so user can know why we are sleeping, then perform the action.
Add timestamp to message so we can measure how long it took here
inform how long we are going to sleep for.

Reply:
message modified.